dist/templates/layout.html

```diff
@@ -0,0 +1,16 @@
+<!doctype html>
```

Comment:
but those files were built by gult 'html' task:
```javascript
gulp.task('html', function () {
    return gulp.src('src/**/*.html')
      .pipe(processhtml({}))
      .pipe(gulp.dest('dist'));
```
wasn't it the aim of this task?